src/crypto/convert.js

```diff
@@ -24,7 +24,18 @@ export const bufferToBigNumberString = bigNumberBuffer =>
 
 export const bufferToHex = buffer => naclInstance.to_hex(buffer);
 
-export const hexToBuffer = hex => Buffer.from(hex, 'hex');
+const hexRegex = /^([0-9]|[a-f])+/gi;
```

Comment:
Let's make this regex as simple as possible. Does the following work?
```
const hexRegex = /^[0-9a-f]+/i;
```

Reply:
i think global key is not needed but `^([0-9]|[a-f])` this is more readable isn't it?

Reply:
As discussed, a capturing group which never gets used is confusing.